### ISSUE TITLE
Wrong version installed with npm

### ISSUE THREAD

**Issue by sullof** [OpenZeppelin/openzeppelin-solidity#974]
## 🎉 Description

Installing `openzeppelin-solidity` using npm does not work as expected with version 1.9.0. 

- [ x ] 🐛 This is a bug report.
- [ ] 📈 This is a feature request.

## 💻 Environment

Not-applicable

## 📝 Details

`0wnable.sol` in version 1.9.0 uses a `constructor()`. But the version installed with npm, even if nominally is the 1.9.0, uses the deprecated `function Ownable()`

## 🔢 Code To Reproduce Issue [ Good To Have ]

If you open a shell and run the following script you can see the difference in the two `Ownable`s.

```
mkdir zeppelin
cd zeppelin
npm init

npm install openzeppelin-solidity
#it shows that the version is the 1.9.0

cat node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol 
# this shows that the contract uses `function Ownable()` as constructor

# we repeat the process installing from github
rm -rf node_modules/

npm i https://github.com/OpenZeppelin/openzeppelin-solidity
# the version is the same

# but Ownable is different
cat node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol 
```

## 👍 Other Information

You can see the output of that script in [this gist](https://gist.github.com/sullof/c26ec489042322bb9cd5a10887d1eb61)

**Comment by sullof** [comment]
Apparently, now it works. Never mind.

**Comment by sullof** [comment]
Sorry, I mistakenly though that it was working but it was so because I didn't remove the folder before running the script again.

**Comment by shrugs** [comment]
fixed in 1.10.0; I have a feeling we incorrectly published 1.9.0 somehow. Thanks for the report!
